mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -194,6 +199,7 @@ class APIRequest:
     convert_chat_responses: bool
     did_perform_chat_conversion: bool
     stream: bool
+    prediction_context: Optional[Context]
```

Comment:
```suggestion
    prediction_context: Optional[Context] = None
```

do we need a default value here?